data/mods/innawood/npcs/missiondef.json

```diff
@@ -0,0 +1,421 @@
+      ]
+    },
+    "end": { "effect": [ "stop_guard", { "set_npc_rule": "investigate_noises" }, { "u_sell_item": "inhaler" }, "npc_thankful" ] },
+    "fail": { "effect": "npc_die" }
+  },
+  {
+    "id": "MISSION_GET_ANTIBIOTICS",
+    "type": "mission_definition",
+    "name": { "str": "Find Cattail Jelly" },
+    "goal": "MGOAL_CONDITION",
+    "difficulty": 2,
+    "value": 150000,
+    "urgent": true,
+    "goal_condition": { "u_has_item": "cattail_jelly" },
+    "origins": [ "ORIGIN_OPENER_NPC" ],
+    "deadline_low": 24,
+    "deadline_high": 48,
+    "dialogue": {
+      "describe": "This infection is bad, <very> bad…",
+      "offer": "I'm infected.  Badly.  I need you to get some antibiotics for me…",
+      "accepted": "Oh, thank god, thank you so much!  I won't last more than a couple of days, so hurry…",
+      "rejected": "What?!  Please, <ill_die> without your help!",
+      "advice": "There's not a town nearby.  I think if you can pull cattail jelly out of some cattails from like a swamp or something, I can eat them.",
+      "inquire": "Find any antibiotics yet?",
+      "success": "Oh thank god!  I'll be right as rain in no time.",
+      "success_lie": "What?!  You're lying, I can tell!  Ugh, forget it!",
+      "failure": "How am I not dead already?!"
+    },
+    "start": {
+      "effect": [
+        { "npc_add_effect": "infection", "duration": "PERMANENT" },
+        { "npc_remove_item_with": "antibiotics" },
+        "assign_guard",
+        { "clear_npc_rule": "investigate_noises" }
+      ]
+    },
+    "end": {
+      "effect": [ { "npc_lose_effect": "infection" }, "stop_guard", { "set_npc_rule": "investigate_noises" }, "npc_thankful" ]
+    },
+    "fail": { "effect": "npc_die" }
+  },
+  {
+    "id": "MISSION_GET_FLAG",
+    "type": "mission_definition",
+    "name": { "str": "Find Rock" },
+    "goal": "MGOAL_FIND_ANY_ITEM",
+    "difficulty": 2,
+    "value": 100000,
+    "item": "rock",
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "Does our flag still yet wave?  Maybe a rock will tell us.",
+      "offer": "Does our flag still yet wave?  We're battered but not yet out of the fight, we need the old colors!  Find me a rock.",
+      "accepted": "Hell ya!  Find me one of those big ol' American flags.  Failing that, just get me a rock to draw on.",
+      "rejected": "Seriously?  God damned commie…",
+      "advice": "Find a large federal building or school, they must have one.  If you can't though, just get me any rock.",
+      "inquire": "Rescued the standard yet?  Or a rock?",
+      "success": "America, fuck ya!",
+      "success_lie": "What?!  You liar!",
+      "failure": "You give up?  This country fell apart because no one could find a good man to rely on… might as well give up, I guess."
+    }
+  },
+  {
+    "id": "MISSION_GET_RECORD_ACCOUNTING",
+    "type": "mission_definition",
+    "name": { "str": "Find Corporate Accounts on Rock" },
```

Comment:
A leather journal?
*Applies to patient records & weather log

Reply:
Done